lib/active_fedora/aggregation/link_inserter.rb

```diff
@@ -12,6 +14,7 @@ def call
       else
         set
       end
+      proxy.container = root.__getobj__
```

Comment:
Can we just use root? I assume you got association errors before, but can you do something like https://github.com/projecthydra-labs/activefedora-aggregation/blob/master/lib/active_fedora/aggregation/ordered_proxy.rb#L13-L15 on the appropriate decorator?